operator/api/v1alpha1/clusterconfig_types.go

```diff
@@ -39,11 +68,27 @@ type ClusterConfigSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
+	// Information about the Workload Plane.
```

Comment:
copy pasta

Reply:
Fixed in https://github.com/scality/metalk8s/pull/4000/commits/827fb5dff969e89c7e4c129e69421e319871fe69